package.json

```diff
@@ -64,6 +64,7 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "5.0.1",
     "react-select": "^5.1.0",
+    "react-toastify": "^9.1.3",
```

Comment:
is there a reason why we're adding an additional dependency instead of using our already existing toast logic?

like here for example
https://github.com/stakwork/sphinx-tribes-frontend/blob/ece9d9844f043852254cfaf1493ce90ac890ac1c/src/people/widgetViews/organization/AddBudgetModal.tsx#L264

Reply:
react-toastify provides a ToastContainer that can be placed in a top-level component (like App.js) and allows to call toast from anywhere in application to display notifications.
and as we the component is being called in another it will increase complexity of the task to manage states
should I still proceed ?

Reply:
please check this file to see how the toasts are done `src/people/widgetViews/organization/AddBudgetModal.tsx`

I don't think it makes sense to have multiple modules that are generating toasts, if we think `react-toastify` is better than `Eui` we should try and migrate them over but we should stick with `EuiGlobalToastList`

Reply:
Okay